docs/using.rst

```diff
@@ -461,5 +461,3 @@ give us as much information as possible:
 - copy and paste ``certbot --version`` output
 - your operating system, including specific version
 - specify which installation method you've chosen
```

Comment:
There is a final newline, there just isn't any trailing empty lines like there was before:
```
$ git checkout os-documentation 
Switched to branch 'os-documentation'
$ python -c "print open('docs/using.rst').read()[-1] == '\n'"
True
```